examples/gptreview-ghaction/README.md

```diff
@@ -0,0 +1,21 @@
+# GPTReview
+
+This folder contains an example of building and implementing your own code reviewer as part of GitHub Actions. 
+
+Below are the files present here:
+
+- `codereview.gpt`: Contains the GPTScript code and prompts.
+- `workflow.yaml`: The workflow file for the GitHub action.
+
+## Pre-requisites
+
+- GitHub Account
+- OpenAI API Key
+
+## How To Run This Example
+
+- Create a new repository in your GitHub account and create a `codereview.gpt` file in that repo based on the contents provided in this file.
+- Congfigure a GitHub Action for that repository and copy the contents from `workflow.yaml` to your `main.yaml`.
+- Configure your `OPENAI_API_KEY` and `GH_TOKEN` as environment variables.
+- Add some code file to the repository and open a new pull request.
```

Comment:
What happens if they push to `main` branch?

Reply:
It should still work, as the action triggers on new pull requests irrespective of the branch. Having said that, I've mentioned to create a new branch and push changes to it.